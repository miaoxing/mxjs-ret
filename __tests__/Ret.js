import {Ret} from '..';
import {render} from '@testing-library/react';
import {Ret as RetObject} from 'miaoxing';

describe('Ret', () => {
  test('basic', () => {
    const result = render(<Ret ret={RetObject.err('error')}/>);
    expect(result.container).toMatchSnapshot();
  });
});
