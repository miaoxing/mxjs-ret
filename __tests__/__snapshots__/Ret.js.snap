// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ret basic 1`] = `
<div>
  <div
    class="css-3waj2m"
  >
    <img
      class="chakra-image__placeholder css-90d413"
    />
    <h2
      class="chakra-heading css-1835qym"
    >
      error
    </h2>
  </div>
</div>
`;
